components/brave_vpn/brave_vpn_service.h

```diff
@@ -197,9 +196,7 @@ class BraveVpnService :
 
   BraveVPNOSConnectionAPI* GetBraveVPNConnectionAPI() const;
 
-  void set_mock_brave_vpn_connection_api(BraveVPNOSConnectionAPI* api) {
-    mock_connection_api_ = api;
-  }
+  void set_mock_brave_vpn_connection_api(BraveVPNOSConnectionAPI* api);
```

Comment:
I think `SetMockBraveVPNConnectionAPI()` would be more fit for chromium convention.
AFAIK, snake case is used for simple setter/getter.

Reply:
done